examples/fluids/vortexshedding.yaml

```diff
@@ -0,0 +1,54 @@
+problem: newtonian
```

Comment:
This now works as a generic model (any domain and BCs), accepting a reference state (initial condition) and freestream state (inherits from reference). In this example, reference is still, but freestream has velocity [1, 0, 0].